act/discovery/__init__.py

```diff
@@ -18,5 +18,6 @@
         'noaapsl': ['download_noaa_psl_data'],
         'neon': ['get_neon_site_products', 'get_neon_product_avail', 'download_neon_data'],
         'surfrad': ['download_surfrad_data'],
+        'nasa' : ["download_mplnet_data", "get_mplnet_meta"]
```

Comment:
I believe there are some linting errors here.  Did you run flake8 on the code at all?

Reply:
I did and came back fine using `python -m flake8 --max-line-length=127 --ignore=F401,E402,W504,W605,F403`

Pre-commit and ruff must be stricter linters

Reply:
I also installed `pre-commit` after I had made commits, when the repo mentioned I should check the updated contributors guide prior to this PR. So I am sure that's how they were missed